View/Screens/RoleAssignmentScreen.xaml.cs

```diff
@@ -45,8 +45,12 @@ private void UserRole_SelectionChanged(object sender, SelectionChangedEventArgs
         {
             User user = (User)((ContentPresenter)((ComboBox)sender).TemplatedParent).Content;
             int roleID = ((ComboBox)sender).SelectedIndex + 1;
+            var artistController = ArtistController.Create(DatabaseContext.Instance);
             if(user.RoleID != roleID)
             {
+                if (user.RoleID == 1) artistController.CreateItem(new Artist {ArtistName = user.Username, UserID = user.ID});
```

Comment:
moet dit niet `user.RoleID != 1` zijn?

Reply:
nope, dit als je user bent en naar artist of admin gaat
dan krijg je een entry in de artist tabel

Reply:
Maar dan vervolgens, als je roleid 1 is, revoke artist?

Reply:
user.roleID is wat je bent, roleID is wat je wordt

Reply:
check, gotcha. Noice

Reply:
![Facebook_logo_thumbs_up_like_transparent](https://user-images.githubusercontent.com/46564984/102618101-7adacb80-413a-11eb-9b05-ff10b5044d49.png)